cluster-scope/components/resourcequotas/bronze/resourcequotas.yaml

```diff
@@ -0,0 +1,12 @@
+kind: ResourceQuota
+apiVersion: v1
+metadata:
+  name: quota
```

Comment:
Please rename to `bronze` (and the rest of quotas accordingly) it we're sticking with these ugly names...

User should be able to tell what tier they've got.

Reply:
> I very much dislike the names - what we're playing on here? Like.. gold, silver, bronze lol. smile 
I believe that's what PSI uses (that's where I got it from). 

I don't really care what we name them, I just want quotas -- I'll update them to small/med/large.